src/i2c.rs

```diff
@@ -316,20 +327,26 @@ macro_rules! hal {
                     self.i2c.cr1.modify(|_, w| w.pe().set_bit());
                 }
 
+                /// Perform an I2C software reset
                 fn reset(&mut self) {
                     self.i2c.cr1.write(|w| w.pe().set_bit().swrst().set_bit());
                     self.i2c.cr1.reset();
                     self.init();
                 }
 
+                /// Generate Start condition
```

Comment:
```suggestion
                /// Generate start condition
```

Reply:
Actually I'd prefer an all uppercase name for conditions to match literature, so START and STOP.

Reply:
That's a good point. I have updated it in the whole file.